src/main/kotlin/no/nav/k9punsj/integrasjoner/dokarkiv/DokarkivGateway.kt

```diff
@@ -250,9 +250,7 @@ class DokarkivGateway(
             .let { journalpost ->
                 val avsenderMottaker = journalpost.getJSONObject("avsenderMottaker")
                 val avsenderIdType = avsenderMottaker.stringOrNull("idType")
-                val avsenderId = if (avsenderIdType == "FNR") avsenderMottaker.stringOrNull("id") else null
-                val avsendernavn = if (avsenderId.isNullOrBlank()) avsenderMottaker.stringOrNull("navn") else null
-
+                val avsendernavn = if (avsenderIdType.isNullOrBlank()) avsenderMottaker.stringOrNull("navn") else null
```

Comment:
Det ser ut til å være en type som heter strengverdien "NULL" som du også må sjekke:
https://confluence.adeo.no/display/BOA/Type%3A+AvsenderMottaker

Reply:
Tatt høyde for nå :)